server/text_generation_server/models/custom_modeling/flash_mpt_modeling.py

```diff
@@ -0,0 +1,361 @@
+"""A simple, flexible implementation of a GPT model.
```

Comment:
I don't think this file was supposed to be added?

Reply:
I removed both .